src/main/java/uk/co/ramp/covid/simulation/output/CsvOutput.java

```diff
@@ -14,7 +15,36 @@
 public class CsvOutput {
     private static final Logger LOGGER = LogManager.getLogger(CsvOutput.class);
 
-    public static void writeDailyStats(Appendable out, int startIterID, List<List<DailyStats>> stats) throws IOException {
+    private static final String STATS_FNAME = "out.csv";
+    private static final String INFECTIONS_OVER_TIME_FNAME = "dailyInfections.csv";
+    private static final String DEATHS_OVER_TIME_FNAME = "deaths.csv";
+    private static final String DEATHS_BY_AGE_FNAME = "deathsByAge.csv";
+    
+    private final Path outputDir;
+    private final List<List<DailyStats>> stats;
+    private final int startIteration;
+
+    public CsvOutput(Path outputDir, int startIteration, List<List<DailyStats>> modelStats) {
+        this.outputDir = outputDir;
+        this.startIteration = startIteration;
+        this.stats = modelStats;
+    }
+    
+    public CsvOutput(int startIteration, List<List<DailyStats>> modelStats) {
+        this(null, startIteration, modelStats);
+    }
+    
+    public void writeOutput() {
+        if (outputDir != null) {
+            writeDailyStats();
+            extraOutputsForThibaud();
+            writeDeathsByAge();
+        } else {
+            LOGGER.warn("Trying to output csv files, but no output directory was set");
```

Comment:
This should surely at least be an error, and we should probably throw an exception?

Reply:
(It should never occur, given the way this class is currently used.)

Reply:
Sounds good - let's go with that!